### ISSUE TITLE
This simple setup doesn't work

### ISSUE THREAD

**Issue by VGerris** [dotnet/docs#17048]
$dotnet new console
Creating this template will make changes to existing files:
  Overwrite   vincent.csproj
  Overwrite   Program.cs

Rerun the command and pass --force to accept and create.
$ dotnet new console --force
The template "Console Application" was created successfully.

Processing post-creation actions...
Running 'dotnet restore' on /home/vincent/vincent.csproj...
  Restore completed in 41.49 ms for /home/vincent/vincent.csproj.

Restore succeeded.

$ dotnet run
/usr/share/dotnet/sdk/3.1.101/Microsoft.Common.CurrentVersion.targets(3056,5): error MSB3552: Resource file "**/*.resx" cannot be found. [/home/vincent/vincent.csproj]

The build failed. Fix the build errors and run again.
This was after following the instructions for setting up the SDK.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d49bc05d-7c8e-5f89-aac5-d133225da4d5
* Version Independent ID: 2f831fc9-dfe6-15db-4ec6-00f59c2dacc2
* Content: [.NET Core guide](https://docs.microsoft.com/en-us/dotnet/core/#feedback)
* Content Source: [docs/core/index.md](https://github.com/dotnet/docs/blob/master/docs/core/index.md)
* Product: **dotnet-core**
* GitHub Login: @richlander
* Microsoft Alias: **mairaw**

**Comment by VGerris** [comment]
Kind of to my surprise, the error went away after a reboot. Since I started a fresh terminal, I wonder if there is a less microsofty way of doing this? It would be nice to see that as part of the setup instructions.

**Comment by VGerris** [comment]
if not entirely clear, the link to the page followed was : https://docs.microsoft.com/en-us/dotnet/core/

**Comment by Thraka** [comment]
I'm not sure what is microsofty about the install and quick-new-app instructions. 😄 

Also, I don't know how you installed .NET Core, but perhaps there was a step there that was missed? Almost like your current terminal session didn't have something of .NET Core in path? Unknown.. Also, what OS were you using?

You referenced some setup instructions, which ones did you use?

Cheers!

**Comment by VGerris** [comment]
Hi,

Thank you for the response.
With microsofty I meant that I had to reboot to make it work ;).
I am running Ubuntu 18.04 and followed:
https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1804
to install the SDK.

The problem is solved, but as I stated the setup instructions don't seem to work as provided. so I suggested to add it.
I'm pretty sure this can be easily reproduced.

I did restart my terminal after the installation before I executed the commands.
For some reason, a reboot made things work, I was asking to have whatever step needed to skip rebooting, added to the instructions, because I don't want to reboot my computer for no reason.

If you think that users should do that, I leave that up to you, i'm not going to reopen this or report this again.
Thank you
